lib/active_model/serializer/associations.rb

```diff
@@ -29,7 +28,8 @@ class << base
 
       module ClassMethods
         def inherited(base)
-          base._reflections = self._reflections.try(:dup) || []
```

Comment:
I see below on line 32, that `base._reflections` is set to a dup of `_reflections` there does the duped `_reflections` come from?